DataRepo/views/upload/submission.py

```diff
@@ -1531,14 +1595,53 @@ def extract_autofill_from_peak_annot_details_sheet(self):
             seq_name = loader.get_row_val(row, loader.headers.SEQNAME)
             self.extract_autofill_from_sequence_name(seq_name, seen)
 
+            # We can actually fill in any missing seqnames in this sheet based on the file (and sample header)
+            # columns...
+            sample_header = loader.get_row_val(row, loader.headers.SAMPLEHEADER)
+            file = loader.get_row_val(row, loader.headers.ANNOTNAME)
+            if file in self.none_vals or sample_header in self.none_vals:
+                continue
+            filename = (os.path.split(file))[1]
```

Comment:
Probably clearer to use [`os.path.basename()`](https://docs.python.org/3/library/os.path.html#os.path.basename) here.
```suggestion
            filename = os.path.basename(file)
```